code/modules/mob/living/carbon/human/species/vox.dm

```diff
@@ -185,7 +186,6 @@
 	icon_template = 'icons/mob/human_races/r_armalis.dmi'
 
 	has_organ = list(
-		"heart" =    /obj/item/organ/internal/heart,
```

Comment:
Perhaps a subtype/new type of heart should be used for species that still uses the old crit system. It's strange that they don't have a heart, period.

Reply:
It's not really strange at all that a slime doesn't have a heart, no.